core/block/import/common/objectcreator/objectcreator.go

```diff
@@ -111,6 +111,7 @@ func (oc *ObjectCreator) Create(dataObject *DataObject, sn *common.Snapshot) (*t
 		log.With("objectID", newID).Errorf("failed to update objects ids: %s", err)
 	}
 
+	oc.updateKeys(st, oldIDtoNew)
```

Comment:
Just to confirm. Does not oldIDtoNew map is getting filled by object type ids and not unique keys?